addons/mass_mailing/static/src/js/mass_mailing_mobile_preview.js

```diff
@@ -29,7 +27,7 @@ export class MassMailingMobilePreviewDialog extends Dialog {
 
     _getSourceDocument() {
         return '<!DOCTYPE html><html>' +
-                    '<head>' + this.styleSheets + '</head>' +
+                    document.head.outerHTML +
```

Comment:
is'nt more correct to do something like
```js
import { getBundle } from "@web/core/assets";

.....

const bundle = await getBundle("mass_mailing.iframe_css_assets_edit");
const links = bundle.cssLibs.map((src) => {
    const link = document.createElement("link");
    link.setAttribute("type", "text/css");
    link.setAttribute("rel", "stylesheet");
    link.setAttribute("href", src);
    return link.outerHTML;
});
return `
    <!DOCTYPE html><html>
        <head> ${links.join("")} </head>
        <body> ${this.props.preview} </body>
    </html>
`;
```

because currently we load everything but only the CSS is useful (I think)

What do you think ?

Reply:
Hello @std-odoo, it's probably cleaner, I updated the PR